launchable/commands/subset.py

```diff
@@ -394,20 +380,20 @@ def run(self):
             rows = [
                 [
                     "Subset",
-                    len(output),
+                    len(original_subset),
                     summary["subset"].get("rate", 0.0),
                     summary["subset"].get("duration", 0.0),
                 ],
                 [
                     "Remainder",
-                    len(rests),
+                    len(original_rests),
                     summary["rest"].get("rate", 0.0),
                     summary["rest"].get("duration", 0.0),
                 ],
                 [],
                 [
                     "Total",
-                    len(output) + len(rests),
+                    len(original_subset) + len(original_rests),
```

Comment:
[question]
Are these correct that you have original_* instead output_*?

Reply:
I think yes, because this is a result of the subset response. Also, `--observation` and `--output-exclusion-rules` are options to change output formats.
So, this data should show original response data